### ISSUE TITLE
Some `E` rules are ignored unless explicitly included in `select`

### ISSUE THREAD

**Issue by figchutney** [astral-sh/ruff#7388]
Hey 👋 

## Summary

I've just set up `ruff` (so speedy!) and noticed something peculiar I thought worth asking about.

**TL;DR** - `ruff` appears to ignore a subset of `E` rules (even though they appear to have been implemented according to https://github.com/astral-sh/ruff/issues/2402). Explicitly including one of the ignored rules in `select` in `pyproject.toml` means that `ruff` _does_ consider the rule.

I'll use [`E225`](https://beta.ruff.rs/docs/rules/missing-whitespace-around-operator/) (`missing-whitespace-around-operator`) as an example, but I suspect this applies to some of the other `E` rules.

### With `select = ["E"]`

`pyproject.toml`
```toml
[tool.ruff]
select = ["E"]
```

`foo.py`
```py
foo =5
```

`ruff check .`
```zsh
➜ ruff check .

```
(i.e. no errors reported)

### With `select = ["E225"]`

`pyproject.toml`
```toml
[tool.ruff]
select = ["E225"]
```

`foo.py`
```py
foo =5
```

`ruff check .`
```zsh
➜ ruff check .
src/foo.py:1:5: E225 Missing whitespace around operator
Found 1 error.
```

Is this expected behaviour? Is there a way I can make `ruff` consider all `E` rules instead of a subset of them?

Thanks! 🙏

**Comment by zanieb** [comment]
Hi! These rules are in preview. See [our documentation](https://beta.ruff.rs/docs/preview/) for details.

Using the `--preview` flag will select those rules.

There's a bug in our documentation where the preview indicator is missing. It'll be restored in our next release (fixed in https://github.com/astral-sh/ruff/pull/7341)

**Comment by figchutney** [comment]
Hey - thanks for such a quick reply. Just to confirm, selecting `E` by itself means `ruff` does _not_ check preview rules within that ruleset, but selecting a preview rule explicitly means `ruff` _does_ check the rule. Is that right?

**Comment by zanieb** [comment]
@figchutney not quite.

`E` does not include preview rules unless you include the `--preview` flag then `E225` would be included. You _can_ select `E225` directly without the preview flag because that's how it used to work — however, we've deprecated that functionality and you'll get a warning.

**Comment by figchutney** [comment]
Thanks for the explaination 👌
